bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/menuitem/listener/LanguageItemSelectionListener.java

```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+ * Copyright 2021-2022 Espressif Systems (Shanghai) PTE LTD. All rights reserved.
+ * Use is subject to license terms.
+ *******************************************************************************/
+
+package com.espressif.idf.ui.menuitem.listener;
+
+import org.eclipse.swt.events.SelectionAdapter;
+import org.eclipse.swt.events.SelectionEvent;
+import org.eclipse.swt.widgets.MenuItem;
+import org.eclipse.ui.PlatformUI;
+
+import com.espressif.idf.core.IDFCorePlugin;
+import com.espressif.idf.core.util.EclipseIniUtil;
+
+public class LanguageItemSelectionListener extends SelectionAdapter
+{
+	private static final String LANGUAGE_SWITCH = "-nl";
+	
+	EclipseIniUtil eclipseIniUtil;
+
+	public LanguageItemSelectionListener()
+	{
+		try
+		{
+			eclipseIniUtil = new EclipseIniUtil();
+		}
+		catch (Exception e)
+		{
+			IDFCorePlugin.getPlugin().getLog().error(e.getMessage());
+		}
+	}
+
+	@Override
+	public void widgetSelected(SelectionEvent selectionEvent)
+	{
+		MenuItem currentItem = ((MenuItem) selectionEvent.widget);
+		String code = currentItem.getText().split("-")[1].replaceAll("^[ \t]+|[ \t]+$", "");
```

Comment:
@kolipakakondal we can but it will only remove the space at the end of the string whereas we need to remove from the beginning as well, I used strip() here before this but our github actions were using Java 1.8 where as the strip was introduced with Java 11